ipaserver/install/adtrust.py

```diff
@@ -80,14 +80,14 @@ def retrieve_netbios_name(api):
 
 def set_and_check_netbios_name(netbios_name, unattended, api):
     """
-    Depending if trust in already configured or not a given NetBIOS domain
-    name must be handled differently.
+    Depending on whether trust is already configured or not a given NetBIOS
```

Comment:
Please fix this sentence to "Depending on whether ***a*** trust is already configured or not, the received NetBIOS domain name must be handled differently."